R/onehotencoding.R

```diff
@@ -429,13 +429,23 @@ holidays <- function(countries = "Venezuela", years = year(Sys.Date()), ...) {
   # Further improvement: let the user bring more than +-5 years
 
   results <- NULL
+  message(paste0(
```

Comment:
Can you please enable the `quiet` parameter here so users can turn this message off when quiet = TRUE (as we do with several other functions in `lares`)? @Patrikios Thanks!

Reply:
No problem, added the parameter quiet to function holidays in the follow-up commit. Merge at your own convenience.

Reply:
third commit with some clean up.
On the side note, was it intended that you set up the if else with the same action lieie follows?:
```
    first <- suppressWarnings(as.numeric(as.character(substr(holidays$Date, 1L, 1L))))
    if (!is.na(first[1L])) {
      holidays$Date <- lubridate::dmy(holidays$Date)
    } else {
      holidays$Date <- lubridate::dmy(holidays$Date)
    }
```

did you mean format `mdy` in the second action?

Reply:
Ugh, not intended. That second one should transform an integer date with `as.Date(holidays$Date, origin = "1970-01-01")` instead. Would you mind fixing that before we merge this PR? Thanks again.

Reply:
Not intended. The original idea behind was this:
```
   if (!is.na(first[1])) {
      holidays$Date <- as.Date(holidays$Date, format = "%d %b %Y")
    } else {
      holidays$Date <- as.Date(holidays$Date, format = "%b %d %Y")
    }
```
So we can change to `lubridate::mdy()` instead or with your change it probably is not necessary given it standardizes the dates.

Reply:
All right. Did the work. As well I added the option to pull in the tibble infos about substate (Details) . I for instance look at them Germany substates where the holidays took place. I think it is usefull for users who dwell in countires with substates division.